mix.exs

```diff
@@ -14,13 +16,21 @@ defmodule Wallaby.Mixfile do
      description: "Concurrent feature tests for elixir",
      deps: deps(),
      docs: docs(),
-     test_coverage: [tool: ExCoveralls],
+
+     # Custom testing
+     aliases: ["test.all": ["test", "test.drivers"],
+               "test.drivers": &test_drivers/1],
```

Comment:
Personally I'm a fan that `mix test` executes all tests, as it does now. So, what I'm saying I'd rather have:

`test` --> everything
`test.unit` --> unit test
`test.integration` --> integration tests

dunno how they do it in ecto, but `mix tes t` to test all it can is my general expectation :)